util/tooltool/tooltool.py

```diff
@@ -392,10 +389,9 @@ def validate_manifest(manifest_file):
         else:
             if not f.validate():
                 invalid_files.append(f)
-    if len(invalid_files + absent_files) == 0:
+    if not invalid_files + absent_files:
```

Comment:
This is correct but not as clear as `if not (invalid_files + absent_files):` (I had to look up the precedence to confirm)